geotrellis/geotrellis-site-deployment
# Remove non-deployment resources

# Overview
This PR starts the process of restructuring `geotrellis-site-deployment` to focus solely on deploying the `geotrellis-site` core infrastructure. Git submodules, `docker-compose` setup, and all Scripts to Rule them All besides `infra` have all been removed. Additionally, terraform resources for the Website ECS service and CDN have been removed.

# Notes
In order to avoid accidental infrastructure deletion during `terraform apply`, any resource that's been removed in this PR has also been removed from the Terraform state. I created geotrellis/geotrellis-site#55  to track the names of those resources, so that they can be added back later. As an additional precaution, I also backed up the old state file on S3.

# Testing
- Run `scripts/setup`, ensure provisioning completes successfully. 
- After following the setup instructions in the `README`, from inside the VM, run `scripts/infra plan`, and be sure there are no changes.
- Make sure that the list of un-managed resources in geotrellis/geotrellis-site#55 matches the terraform resources removed in this diff.

Fixes #4
